example/agent.hcl

```diff
@@ -7,3 +7,10 @@ plugin "containerd-driver" {
     stats_interval = "5s"
   }
 }
+
+client {
+  host_volume "s1" {
+    path = "/tmp/host_volume/s1"
+    read_only = false
```

Comment:
Can we change `read_only = true` and add a check in the integration test for `read_only = true` where the check tries to write a file and fails?

Reply:
+1

Reply:
read_only mount is added in example/volume_mount.nomad

```
    volume "read_only_data" {
      type = "host"
      source = "s1"
      read_only = true
    }
```

```
      volume_mount {
        destination = "/tmp/read_only_target"
        volume = "read_only_data"
      }
```

and two checking is added to the test:

```
INFO: Checking read only mount is not writable.
touch: cannot touch '/tmp/read_only_target/writable_test.txt': Read-only file system
INFO: Checking non read_only mount is writable.
```